doc/source/reference/random/multithreading.rst

```diff
@@ -26,17 +26,17 @@ seed will produce the same outputs.
 
     class MultithreadedRNG:
         def __init__(self, n, seed=None, threads=None):
-            rg = PCG64(seed)
+            bg = PCG64(seed)
             if threads is None:
                 threads = multiprocessing.cpu_count()
             self.threads = threads
 
-            self._random_generators = [rg]
-            last_rg = rg
+            self._random_generators = [Generator(bg)]
+            last_bg = bg
             for _ in range(0, threads-1):
-                new_rg = last_rg.jumped()
-                self._random_generators.append(new_rg)
-                last_rg = new_rg
+                new_bg = last_bg.jumped()
```

Comment:
I though we were going to get rid of jumped. 
I think this could be 

```
    class MultithreadedRNG:
        def __init__(self, n, seed=None, threads=None):
            seq = SeedSequence(seed)
            if threads is None:
                threads = multiprocessing.cpu_count()
            self.threads = threads

            self._random_generators = [Generator(PCG64(s)) for s in seq.spawn(threads) ]
...
```

Reply:
done